src/components/user/Hackpack.jsx

```diff
@@ -4,18 +4,22 @@ import Link from "next/link";
 const Hackpack = ({ text, techs, link }) => {
   return (
     <Link
+      data-cy="link-component-test"
```

Comment:
data-cy names should be consistent, should all be of the form `hackpack-*` ie `hackpack-link` `hackpack-text`